scrapers/SLRoriginals.yml

```diff
@@ -0,0 +1,60 @@
+name: "VRBangers"
```

Comment:
Leftover from copy paste? Use the correct name

Reply:
Yeah, sorry. It is a leftover.